### PR TITLE
[zh] Sync Ambient and Sidecar Traffic blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync Ambient and Sidecar Traffic blog into Chinese:
```
content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+## 总结 {#wrapping-up}
+
+Sidecar 模式使 Istio 成为一个出色的服务网格。但是，Sidecar 模式也会导致问题，
+因为它要求应用程序和 Sidecar 容器在同一个 Pod 中运行。
+Istio Ambient 模式通过集中式代理（ztunnel 和 waypoint）实现服务之间的通信。
+Ambient 模式提供了更大的灵活性和可扩展性，减少了资源消耗，因为它不需要网格中的每个 Pod 都有 Sidecar，
+并且允许更精确的配置。 因此，毫无疑问，Ambient 模式是 Istio 的下一个演进。
+显然，Sidecar 和 Ambient 模式的共存可能会持续很长一段时间，
+虽然 Ambient 模式还处于 Alpha 阶段，Sidecar 模式仍然是 Istio 的推荐模式，
+但它将给用户提供更轻量级的选择 随着 Ambient 模式转向测试版和未来版本，运行并采用 Istio 服务网格。
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:145]:
```suggestion
但随着 Ambient 模式进阶至 Beta 和后续发布，它将给用户提供一种更轻量级的选择来运行并采用 Istio 服务网格。
```

**Comment by wilsonwu** [comment]
Yes, I am keeping on my eyes on the fixes and will sync on time.

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+在现代微服务架构中，服务之间的通信和管理至关重要。为了应对这一挑战，
+Istio 作为一种服务网格技术应运而生。它利用 Sidecar 提供流量控制、安全性和卓越的可观察功能。
+为了进一步提高 Istio 的适应性和灵活性，Istio 社区开始探索一种新的模式 —— Ambient 模式。
+在这种模式下，Istio 不再依赖显式的 Sidecar 注入，而是通过 ztunnel 和 waypoint
+代理实现服务之间的通信和网格管理。Ambient 还带来了一系列改进，例如更低的资源消耗、更简单的部署以及更灵活的配置选项。
+启用 Ambient 模式后，我们不再需要重新启动 Pod，这使得 Istio 在各种场景中能够更好地发挥作用。
+
+有很多介绍和分析 Ambient 的博客，
+这些内容可以在本博客的[参考资源](#reference-resources)部分找到，
+本博客将分析 Istio Ambient 和 Sidecar 模式下的网络流量路径。
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:24]:
```suggestion
本文将分析 Istio Ambient 和 Sidecar 模式下的网络流量路径。
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+在现代微服务架构中，服务之间的通信和管理至关重要。为了应对这一挑战，
+Istio 作为一种服务网格技术应运而生。它利用 Sidecar 提供流量控制、安全性和卓越的可观察功能。
+为了进一步提高 Istio 的适应性和灵活性，Istio 社区开始探索一种新的模式 —— Ambient 模式。
+在这种模式下，Istio 不再依赖显式的 Sidecar 注入，而是通过 ztunnel 和 waypoint
+代理实现服务之间的通信和网格管理。Ambient 还带来了一系列改进，例如更低的资源消耗、更简单的部署以及更灵活的配置选项。
+启用 Ambient 模式后，我们不再需要重新启动 Pod，这使得 Istio 在各种场景中能够更好地发挥作用。
+
+有很多介绍和分析 Ambient 的博客，
+这些内容可以在本博客的[参考资源](#reference-resources)部分找到，
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:23]:
```suggestion
在本文的[参考资源](#reference-resources)一节列出了很多在社区和技术论坛中介绍和分析 Ambient 的博客，
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+本博客将分析 Istio Ambient 和 Sidecar 模式下的网络流量路径。
+
+为了阐明网络流量路径并使其更容易理解，本博文探讨了以下两种场景并配有相应的图表：
+
+- **Ambient 模式下的服务到 Sidecar 模式下服务的网络路径**
+- **Sidecar 模式下的服务到 Ambient 模式下服务的网络路径**
+
+## 关于分析的信息 {#information-about-the-analysis}
+
+分析基于 Istio 1.18.2，其中 Ambient 模式使用 iptables 进行重定向。
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:33]: line 31-33 has different content with en now:

```
_Note 1: The following analysis is based on Istio 1.18.2, where ambient mode uses iptables for redirection._

_Note 2: The communications between sidecar and ztunnel/waypoint proxy uses `[HTTP Based Overlay Network (HBONE)](https://docs.google.com/document/d/1Ofqtxqzk-c_wn0EgAXjaJXDHB9KhDuLe-W3YGG67Y8g/edit)`._
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+## Ambient 模式 `sleep` 到 Sidecar 模式 `httpbin` {#ambient-mode-sleep-to-sidecar-mode-httpbin}
+
+### 第一个场景的部署和配置 {#deployment-and-configuration-for-the-first-scenario}
+
+- `sleep` 部署在名称空间 foo 中
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:39]:
```suggestion
- `sleep` 部署在命名空间 foo 中
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+{{< image width="100%"
+    link="ambient-to-sidecar.png"
+    caption="Ambient 模式 sleep 到 Sidecar 模式 httpbin"
+    >}}
+
+如果启用 Ambient 模式，ztunnel 将作为 DaemonSet 部署在 istio-system 命名空间中，
+而 istio-cni 和 ztunnel 将为 ztunnel Pod 和每个节点上的 Pod 生成 iptables 规则和路由。
+
+启用 Ambient 模式的 Pod 进出的所有网络流量都将根据网络重定向逻辑通过 ztunnel。
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:56]:
```suggestion
启用 Ambient 模式时进出 Pod 的所有网络流量都将根据网络重定向逻辑流经 ztunnel。
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+然后 ztunnel 会将流量转发到正确的端点。
+
+### Ambient 模式 `sleep` 到 Sidecar 模式 `httpbin` 的网络流量路径分析 {#network-traffic-path-analysis-of-ambient-mode-sleep-to-sidecar-mode-httpbin}
+
+根据上图，网络流量路径的详细情况如下所示：
+
+**(1) (2) (3)** `sleep` 服务的请求流量从 `sleep` Pod 的 `veth` 发出，
+在该 Pod 中被标记并转发到 `istioout` 设备。节点遵循 iptables 规则和路由规则。
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:64]:
```suggestion
遵循 iptables 规则和路由规则该 Pod 被标记并转发到节点中的 `istioout` 设备。
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+节点 A 中的 `istioout` 设备是一个 [Geneve](https://www.rfc-editor.org/rfc/rfc8926.html) 隧道，
+隧道的另一端是 `pistioout`，即位于同一节点上的 ztunnel Pod 内。
+
+**(4) (5)** 当流量通过 `pistioout` 设备到达时，
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:68]:
```suggestion
**(4) (5)** 当流量经由 `pistioout` 设备到达时，
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+Pod 内的 iptables 规则会拦截并通过 Pod 中的 `eth0` 接口将其重定向到端口 `15001`。
+
+**(6)** 根据原始请求信息，ztunnel 可以获取目标服务的端点列表。
+然后，它将处理将请求发送到端点，例如 `httpbin` Pod 之一。
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:72]:
```suggestion
然后，它将处理请求并将其发送到端点，例如某个 `httpbin` Pod。
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 10]
```diff
@@ -0,0 +1,151 @@
...
+规则拦截并重定向到 Sidecar 的端口 `15006`。
+
+**(8)** Sidecar 处理通过端口 15006 传入的入站请求流量，
+并将流量转发到同一 Pod 中的 `httpbin` 容器。
+
+## Sidecar 模式 `sleep` 到 Ambient 模式 `httpbin` 和 `helloworld` {#sidecar-mode-sleep-to-ambient-mode-httpbin-and-helloworld}
+
+### 第二种场景的部署和配置 {#deployment-and-configuration-for-the-second-scenario}
+
+- `sleep` 部署在名称空间 foo 中
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:85]:
```suggestion
- `sleep` 部署在命名空间 foo 中
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 11]
```diff
@@ -0,0 +1,151 @@
...
+{{< image width="100%"
+    link="sidecar-to-ambient.png"
+    caption="sleep 到 httpbin 和 helloworld"
+    >}}
+
+### Sidecar 模式 `sleep` 到 Ambient 模式 `httpbin` 的网络流量路径分析 {#network-traffic-path-analysis-of-sidecar-mode-sleep-to-ambient-mode-httpbin}
+
+上图的上半部分描述了从 `sleep` Pod（Sidecar 模式）到 `httpbin` Pod（Ambient 模式）的请求的网络流量路径。
+
+**(1) (2) (3) (4)** `sleep` 容器向 `httpbin` 发送请求。该请求被 iptables 规则拦截，
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:108]:
```suggestion
**(1) (2) (3) (4)** `sleep` 容器向 `httpbin` 发送一个请求。该请求被 iptables 规则拦截，
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 12]
```diff
@@ -0,0 +1,151 @@
...
+并定向到 `sleep` Pod 中 Sidecar 上的端口 `15001`。
+然后，Sidecar 处理请求并根据从 istiod（控制平面）收到的配置路由流量，
+将流量转发到与节点 B 上的 `httpbin` Pod 对应的 IP 地址。
+
+**(5) (6)** 将请求发送到设备对（`veth httpbin <-> eth0 inside httpbin pod`）后，
+请求被拦截并使用 iptables 和路由规则转发到 `istioin` 通过遵循其 iptables 和路由规则，
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:114]:
```suggestion
请求根据 iptables 和路由规则被拦截和转发到节点 B 上的 `istioin` 设备，
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 13]
```diff
@@ -0,0 +1,151 @@
...
+运行 `httpbin` Pod 的节点 B 上的设备。节点 B 上的 `istioin` 设备和同一节点上
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:115]:
```suggestion
在节点 B 上遵循 iptables 和路由规则来运行 `httpbin` Pod。节点 B 上的 `istioin` 设备和同一节点上
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 14]
```diff
@@ -0,0 +1,151 @@
...
+然后 ztunnel 会将请求转发到同一节点 B 中的 `httpbin` Pod。
+
+### 通过 waypoint 代理从 Sidecar 模式 `sleep` 到 Ambient 模式 `httpbin` 的网络流量路径分析 {#network-traffic-path-analysis-of-sidecar-mode-sleep-to-ambient-mode-httpbin-via-waypoint-proxy}
+
+与图中的顶部相比，底部在 `sleep`、ztunnel 和 `httpbin` Pod 之间的路径中插入了一个 waypoint 代理。
+Istio 控制平面拥有服务网格的所有服务和配置信息。当使用 waypoint 代理部署 `helloworld` Pod 时，
+`sleep` Pod 的 Sidecar 接收到的 `helloworld` 服务的 EDS 配置将更改为 `envoy_internal_address` 类型。
+这会导致通过 Sidecar 的请求流量通过
+[基于 HTTP 的覆盖网络（HBONE）](https://docs.google.com/document/d/1Ofqtxqzk-c_wn0EgAXjaJXDHB9KhDuLe-W3YGG67Y8g/edit)
+协议转发到节点 C 上的 waypoint 代理的 15008 端口。
```
Comment (windsonsea) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:131]:
```suggestion
协议被转发到节点 C 上的 waypoint 代理的 15008 端口。
```

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md (hunk 4 above)
Comment (wilsonwu) [reply content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:33]: Yes, let's review after the fixes PR merged.

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md [hunk 15]
```diff
@@ -0,0 +1,151 @@
+---
+title: "深挖 Ambient 与 Sidecar 并存的网络流量路径"
+description: "深挖 Ambient 与 Sidecar 并存的流量路径。"
+publishdate: 2023-09-18
+attribution: "Steve Zhang (Intel), John Howard (Google), Yuxing Zeng(Alibaba), Peter Jausovec(Solo.io); Translated by Wilson Wu (DaoCloud)"
```
Comment (zhlsunshine) [inline content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:5]: Hi @wilsonwu, could you please help to replace with the the Chinese names for Yuxing and me? Thanks! ^_^

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md (hunk 4 above)
Comment (wilsonwu) [reply content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:32]: All synced.

File: content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md (hunk 15 above)
Comment (wilsonwu) [reply content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:5]: Yes, no problem, could you please give me the correct Chinese name characters of you guys? Thanks.

Comment (zhlsunshine) [reply content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:5]: @wilsonwu Yes, sure, `Steve Zhang` ---> `张怀龙` and `Yuxing Zeng` ---> `曾宇星`

Comment (wilsonwu) [reply content/zh/blog/2023/traffic-for-ambient-and-sidecar/index.md:5]: Done

**Comment by wilsonwu** [comment]
The fixes from #13912 synced.

**Review by windsonsea — APPROVED** [review]
Thanks.
/lgtm

**Comment by wilsonwu** [comment]
> In response to a cherrypick label: failed to push cherry-picked changes in GitHub: pushToNamedFork is not implemented in the v2 client

@ericvn please take a look for this, I just want to let this Chinese version blog published.

**Pull request merged**